src/app.py

```diff
@@ -43,7 +46,7 @@ def load_user(id):
 
 @app.route('/')
 def index():
-	return render_template('index.html', commands=json.dumps(commands_list), prompt=get_prompt())
+	return render_template('index.html', commands=json.dumps(commands_list), prompt=json.dumps(get_prompt())[1:-1])
```

Comment:
I don't understand this one. get_prompt() returns a string, so json.dumps is going to spit out the same string except with it quoted? is that what the [1:-1] is for? To remove the quotes. 

You should be able to just stick get_prompt() here.

Reply:
After changing the get path function to use string literals it seems to have solved my escaping issue (because of the \'s in the paths) that caused me to add the jsonify here. Done removed.